src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EvpPkey.cs

```diff
@@ -105,18 +105,28 @@ internal static partial class Crypto
         }
 
         [GeneratedDllImport(Libraries.CryptoNative)]
-        private static partial int CryptoNative_GetPkcs8PrivateKeySize(IntPtr pkey);
+        private static partial int CryptoNative_GetPkcs8PrivateKeySize(IntPtr pkey, out int p8size);
 
         private static int GetPkcs8PrivateKeySize(IntPtr pkey)
         {
-            int ret = CryptoNative_GetPkcs8PrivateKeySize(pkey);
+            const int Error = -1;
+            const int MissingPrivateKey = -2;
+            const int Success = 1;
```

Comment:
Nit: declare these in the same order their described for the native header (it's obviously super minor, it was just a little jarring to me when comparing the lists)
```suggestion
            const int Success = 1;
            const int Error = -1;
            const int MissingPrivateKey = -2;
```